plasTeX/Imagers/pdf2svg.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import subprocess
+import re
+import xml.etree.ElementTree as ET
+from plasTeX.Imagers import VectorImager
+
+length_re = re.compile('([0-9\\.]*)(.*)')
+class PDFSVG(VectorImager):
+    """ Imager that uses pdf2svg """
+    fileExtension = '.svg'
+    verifications = ['pdflatex --help', 'which pdf2svg', 'pdfcrop --help']
+    compiler = 'pdflatex'
+
+    def executeConverter(self, output):
+        open('images-uncropped.pdf', 'wb').write(output)
```

Comment:
I know plasTeX is full of code like this. But it was written 15 years ago, let's not add more. Please use 
```python
with open('images-uncropped.pdf', 'wb') as uncropped:
    uncropped.write(output)
```
which makes sure to close the file handler immediately, or even more modern: start the file with `from pathlib import Path` and then use `Path('images-uncropped.pdf').write_bytes(output)` instead of the above line, since the next plasTeX release will support only python at least 3.5.

Reply:
Glad to hear about the last part. Does that mean I'm allowed to add type hints as I go?

Reply:
Yes! You can add as many type hints as you wish, our lives would be much easier with type hints in plasTeX. Even without using mypy, it's already great documentation. But let's keep it python 3.5 compatible for now (so no PEP 526 variable type hints).